frontend/device/kindle/device.lua

```diff
@@ -160,6 +160,8 @@ local Kindle = Generic:extend{
     isNightModeChallenged = no,
     -- NOTE: While this ought to behave on Zelda/Rex, turns out, nope, it really doesn't work on *any* of 'em :/ (c.f., ko#5884).
     canHWDither = no,
+    -- Device has an Ambient Light Sensor
+    hasLightSensor = no,
```

Comment:
Shouldn't that also go in generic device?

Reply:
It could, but nobody's been brave enough to actually successfully implement an ALS outside of lab126 ;p.

Reply:
Fair enough.

Reply:
(The implementation used by the plugin in question is also very much Kindle-only, so that wouldn't even allow simplifying the plugin's check).